### DISCUSSION TITLE
What is the best threshold?

### DISCUSSION THREAD

**Discussion by JoostvanBruggen** [Mattk70/Chirpity-Electron#94] [General]
I can't seem to find out what the best threshold is for my area (Netherlands) when I put the list on Local Birds? Sometimes it can't find a recorded Spotted Crake at a certain level and when I change the list to Birds and tweek different options the there it is again.
But now the Water rails are 50% gone.
Does this mean I have to use several detection sttings to get alle the birds? It seems so but I can't beleive that. Tough there doesn't seem to be just one ideal setting.. unfortunatly. Forcing people to constantly tweek and change settings, no?

**Comment by Mattk70** [comment]
The logic behind the local list is not something I fully understand! It uses the BirdNET list model to infer what birds may be seen locally, and this is based on eBird submissions (and consequently, not great for nocmig). There's a discussion piece on the BirdNET method to caclulate local birds written by Stefan Kahl here: https://github.com/kahst/BirdNET-Analyzer/discussions/234. I have had similar issues with the lists it gives and posted a question about that here: https://github.com/kahst/BirdNET-Analyzer/issues/296 

If you know what species should be included, or excluded, the best way is to create a custom list. That way you have precise control over the species that are reported. You could also use different custom lists for different times of year. N.B. you need to set up your list separately for the Chirpity and BirdNET models, as they use different classes / class names. 

The quickest way to create a custom list is to use the local birds settings to generate a good approximate list, export this list from the "What species are detected" link and add / remove the species you want to include from this.

**Reply by JoostvanBruggen** [reply]
I'll try and make my own list! Thank you!
